docs/devguide/create-metricset.asciidoc

```diff
@@ -242,12 +242,19 @@ var (
 )
 
 func eventMapping(input map[string]interface{}) common.MapStr {
-	return schema.Apply(input)
+	return schema.Apply(input) <4>
 }
 ----
+<1> `Required` will explicitly mark a field as non-optional.
+<2> If a field has no Schema Option set, it is equivalent to `Required`.
+<3> `Optional` You can also explicitly mark a field as not required.
+<4> By default, `Apply` will fail and return an error if any field not marked as optional is missing. Using the optional second argument, you can specify how `Apply` handles different fields of the  schema. The possible values are:
+- `AllRequired` is the default behavior. Return an error if any field not marked as optional is missing.
```

Comment:
If you want to disambiguate further, you could add "...any required field is missing, including fields that are required because no schema option is set."

Reply:
Yah, that's what I was trying to get across. Thanks!